README.md

```diff
@@ -133,4 +133,105 @@ Because the switching of build targets is time-intensive, you can set up **multi
 ## Data Attribution
 Star data sourced from the [HYG Database](https://astronexus.com/hyg)
 https://github.com/astronexus/HYG-Database
-Constellation Lines from [Stellarium](https://github.com/Stellarium/stellarium/blob/master/skycultures/western/constellationship.fab)
\ No newline at end of file
+Constellation Lines from [Stellarium](https://github.com/Stellarium/stellarium/blob/master/skycultures/western/constellationship.fab)
+
+
+# Internal researcher documentation:
+## How to collect anonymous log data from the Concord Log Puller:
+
+1. You will need the HMAC secret from the Portal used by the Log Puller. You can find this key by searching for "Log Puller HMAC Secret from Portal" in the CC 1Password vault called "Shared".
+2. Open a browser to the raw [Log Manager interface](https://apps.concord.org/log-puller/portal-report)
+5. Expand the red help text by clicking on the blue "click here" text. This will give you more information about how to structure your query, and how to use your secret HMAC Token.
+4. Enter your log query. Only search for very limited time-range. 24 hours or less I would suggest. See item 10, or the example below for what your query should look like.
+7. Choose your format, `json` or, more likely `csv`.
+8. Select `Expand Parameters and Extras in CSV` to get additional useful CSV columns.
+7. Enter your HMAC secret into the first wide text field (see screenshot).
+8. Click "Count Logs" to get a sense of how many records will be returned by your query.
+9. When you are satisfied with that number, you can click "Download Logs".
+10. To construct a query, ensure the `application` is set to "CEASAR" and be sure to include a `time` range. You can use this query as a template, but please change the `time` expression.
+
+```
+{
+  "query": [
+    "and",
+    ["=", "application", "CEASAR"],
+    [">=", "time", "2019-12-20"]
```

Comment:
Maybe set this to a future date so that if anyone forgets to change the date they get no logs rather than a bazillion logs.

Reply:
Yes, I think that is a great idea.